index.html

```diff
@@ -17,205 +17,228 @@
+              <li>Gitlab</li>
+              <li>Terminal</li>
             </ul>
-          </hrx>    
-          <button type="button">See Project</button>
-        </hrx>
       </section>
-    </section>
+    </main>
   
-    <section class="about-us">
-      <h2 class="primary-text">About me</h2>
-      <p class="description">Hello I’m a software developer! I can help you build a product, 
-        feature or website Look through some of my work and experience! 
-        If you like what you see and have a project you need coded, 
-        don’t hestiate to contact me.</p>
-        <button type="button">Get My Resume</button>
-    </section>   
-
-    <section class="skills-container skills-category">
-      <hxr class="skils-title">
-        <img src="images/diomand.png" alt="Diomand shape">
-        <h3 class="language">Language</h3>
-      </hxr>
-        <ul>
-          <li>Javascript</li>
-          <li>Ruby</li>
-          <li>HTML</li>
-          <li>CSS</li>
-        </ul>
-
-        <hxr class="skils-title">
-          <img src="images/rectangle.png" alt="Diomand shape">
-          <h3 class="language">Frameworks</h3>
-        </hxr>
-          <ul>
-            <li>Bootstrap</li>
-            <li>Ruby on Rails</li>
-            <li>RSPec</li>
-            <li>CapyBara</li>
-            <li>Selenium</li>
-          </ul>
-
-        <hxr class="skils-title">
-          <img src="images/circle.png" alt="Diomand shape">
-          <h3 class="language">Skills</h3>
-        </hxr>
+    <footer> 
+      <section class="contact">
+        <p>I'm always interested in hearing about new projects,
+        so if you'd like to chat please get in touch.</p>
+        <form action="https://formspree.io/f/{form_id}" method="post">
+          <input type="text" id="fullname" placeholder="Full Name" maxlength="30" required>
+          <input type="email" id="email" placeholder="Email address" required>
+          <textarea id="message" maxlength="50" placeholder="Enter text here" required></textarea>
+          <button type="submit" id="get-in-touch" >Get in touch</button>
+        </form>
+      </section>
+      <section class="bottom_nav">
+          <img id="breakline" src="images/breakline.png" alt="Home Indicator/Light">
           <ul>
-            <li>Codekit</li>
-            <li>Github</li>
-            <li>Codepen</li>
-            <li>Gitlab</li>
-            <li>Terminal</li>
+            <li><a href="https://github.com/cherelemma/"><img src="images/frame1.png" alt="Github Icon"></a></li>
+            <li><a href="https://www.linkedin.com/in/chere-lemma27211613/"><img src="images/frame2.png" alt="Linkedin Icon"></a></li>
+            <li><a href="#"><img src="images/frame3.png" alt="Angel list Icon"></a></li>
+            <li><a href="https://twitter.com/Chere21271613"><img src="images/frame4.png" alt="Twitter Icon"></a></li>
+            <li><a href="#"><img src="images/frame5.png" alt="Medium Icon"></a></li>
           </ul>
```

Comment:
- [ ] Kindly consider using the correct path for your social icon as you can see how they are displayed on your page

Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/199579723-6ff1f209-260c-4e14-a451-517aa3e6f77f.png)